.nvmrc

```diff
@@ -1 +1 @@
-14.17.0
+18.8.0
```

Comment:
According to the node release guidance 'Production applications should only use Active LTS or Maintenance LTS releases' and v18 doesn't technically enter active LTS until 2022-10-25. I guess it depends on how that is interpreted given Canopy is a UI library but it is ultimately used by production applications so my interpretation is that we should use v16.x.x for the next month and a bit

Reply:
link to node releases https://nodejs.org/en/about/releases/

Reply:
That's a good point, although there seem to be no issue running with this version. Also, the fact that the move to active status is only in a month makes me think most libraries have already upgraded to it 🤔 

```
Major Node.js versions enter Current release status for six months, which gives library authors time to add support for them
```

@pauleustice do you have an opinion on this since we made the same change in the monorepo too?